lib/cryptor.rb

```diff
@@ -0,0 +1,11 @@
+class Cryptor
+
+  attr_reader :message, :key, :date
+
```

Comment:
the `Cryptor` class will include methods that are common to the `Encryptor` and `Decryptor` classes.